readme.md

```diff
@@ -0,0 +1,161 @@
+# Remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+**[remark][]** is a tool that transforms markdown with plugins.
+These plugins can inspect and change your markup.
+You can use remark on the server, the client, CLIs, deno, etc.
+This plugin for Visual Studio Code lets you format and lint markdown files using
+remark.
+
+## Contents
+
+*   [When should I use this?](#when-should-i-use-this)
+*   [Use](#use)
+    *   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## When should I use this?
+
+If your project is checked using [`remark-cli`][remark-cli] and you use Visual
+Studio Code to edit markdown files, it’s recommended to use this extension.
+
+## Use
+
+To start using this for a project, [`remark-cli`][remark-cli] needs to be
+configured first.
+For example, install `remark-cli` and `remark-preset-lint-recommended`:
+
+```sh
+npm install remark-cli remark-preset-lint-recommended
+```
+
+Now create a file named `.remarkrc.json` with the following content:
+
+```json
+{
+  "plugins": [["preset-lint-recommended"]]
+}
+```
+
+Now any syntax which doesn’t comply with `remark-preset-lint-recommended` will
+be reported in the *Problems* pane in your editor.
```

Comment:
Should this explain that a user should open a readme or whatever? Perhaps good to start with a markdown example, and then here show what the output in `Problems` is, if everything is working perfectly